src/main/webapp/js/common.js

```diff
@@ -169,8 +169,15 @@ function isTouchDevice() {
  *     The column index (0-based) e.g. use 2 if <th> has 2 rows so that the headers are not sorted.
  */
 function toggleSort(divElement, colIdx, comparator, row) {
```

Comment:
The 4th parameter `row` seems unused. If that's really the case, you can remove it altogether.
While you're at it, also update the header comment of this method to sync with the new parameters.